api/outputs.js

```diff
@@ -0,0 +1,47 @@
+import apiRequest from './index'
+
+const fetchMetadata = async (id) => {
+  // TODO: Centralise API_URL constructing
+  const url = new URL(`/v3/outputs/${id}`, process.env.API_URL).href
+  const { data } = await apiRequest(url)
+  return data
+}
+
+const fetchSimilarTo = async (id) => {
+  const url = new URL('/v3/recommend', process.env.API_URL)
+  const body = { identifier: `core:${id}` }
```

Comment:
doesn't require a fix here probably, but i think we should reduce the number of recommendations to the top 5 for two main reasons:
1. the accuracy after the 5 might. get poor
2. the recommendations are taking too much space on the page, especially when there is no too much abstract

Reply:
I can put a number. It's quite easy but let me think where it's best to do so.

Reply:
Let's add a `limit:5` to the API call for now?
```suggestion
  const body = { identifier: `core:${id}` , limit:5}
```

Reply:
I will take this into account.